MAUI/TimePicker/customizations.md

```diff
@@ -0,0 +1,403 @@
+The Default value of [OkButtonText] property is "OK", [CancelButtonText] is "Cancel". To enable the footer view by setting the [Height] property grater than 0 of [PickerFooterView]. The default value of `Height` property is 0.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+<picker:SfTimePicker x:Name="Timepicker" >
+    <picker:SfTimePicker.FooterView >
+        <picker:PickerFooterView Height="40" OkButtonText="Ok Button"
+                                 CancelButtonText="Cancel Button"/>
+    </picker:SfTimePicker.FooterView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C#" %}
+
+SfTimePicker timePicker = new SfTimePicker();
+timePicker.FooterView = new PickerFooterView()
+{
+    Height = 40,
+    OkButtonText = "Ok Button",
+    CancelButtonText = "Cancel Button",
+};
+
+this.Content = timePicker;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Set divider color
+
+The SfTimePicker control allows you to customize the footer divider color by setting the [DividerColor] property of [PickerFooterView].
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+<picker:SfTimePicker x:Name="Timepicker" >
+    <picker:SfTimePicker.FooterView >
+        <picker:PickerFooterView DividerColor="#A9A9A9" />
+    </picker:SfTimePicker.FooterView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C#" %}
+
+SfTimePicker timePicker = new SfTimePicker();
+timepicker.FooterView = new PickerFooterView()
+{
+    DividerColor = Color.FromArgb("#A9A9A9")
+};
+
+this.Content = timePicker;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Customization of footer
+
+You can customize the footer text style and background color of the `Time Picker` using the [TextStyle] and [Background] properties of the [PickerFooterView].
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+<picker:SfTimePicker x:Name="Timepicker" >
+    <picker:SfTimePicker.FooterView >
+        <picker:PickerFooterView Background="#6750A4">
+            <picker:PickerFooterView.TextStyle >
+                <picker:PickerTextStyle FontSize="15" TextColor="White" />
+            </picker:PickerFooterView.TextStyle>
+        </picker:PickerFooterView>
+    </picker:SfTimePicker.FooterView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C#" %}
+
+SfTimePicker timePicker = new SfTimePicker();
+timePicker.FooterView = new PickerFooterView()
+{
+    Background = Color.FromArgb("#6750A4"),
+    TextStyle = new PickerTextStyle()
+    {
+        TextColor = Colors.White,
+        FontSize = 15,
+    }
+};
+
+this.Content = timePicker;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Selection View Customization
+
+You can customize the time picker selection view by using the `SelectionView` property of `SfTimePicker`.
+
+### Set selection view
+
+In the SfTimePicker control, corner radius, stroke , and padding can be customized by setting the [CornerRadius], [Stroke] and [Padding] properties in the [PickerSelectionView].
```

Comment:
In the SfTimePicker control, the corner radius, stroke, and padding can be customized by setting the [CornerRadius], [Stroke], and [Padding] properties in the [PickerSelectionView].

Reply:
Comments modified.